### PR TITLE
add action queue and update usage

### PR DESCRIPTION
**Pull request by hopeyen**
Update documentation on indexer components

### CODE REVIEW

**Comment by azf20** [comment]
hey thanks @hopeyen! Could you possibly rebase against the new docs structure? (the indexer docs are still one page, just here https://github.com/graphprotocol/docs/blob/main/pages/en/network/indexing.mdx)

**Comment by hopeyen** [comment]
hi @azf20 yep, rebased :D

File: pages/en/network/indexing.mdx [hunk 1]
```diff
@@ -558,25 +571,135 @@ Data model:
...
+graph indexer rules offchain QmZfeJYR86UARzp9HiXbURWunYgC9ywvPvoePNbuaATrEK
+
+graph indexer rules set QmZfeJYR86UARzp9HiXbURWunYgC9ywvPvoePNbuaATrEK decisionBasis always allocationAmount 123321 allocationLifetime 14 autoRenewal false requireSupported false
+
+graph indexer rules stop QmZfeJYR86UARzp9HiXbURWunYgC9ywvPvoePNbuaATrEK
+
+graph indexer rules delete QmZfeJYR86UARzp9HiXbURWunYgC9ywvPvoePNbuaATrEK
+```
+
+#### Actions queue CLI
```
Comment (azf20) [inline pages/en/network/indexing.mdx:615]: Maybe worth mentioning the version here too

**Review by azf20 — APPROVED** [review]
These look good to me! Some small notes

File: pages/en/network/indexing.mdx [hunk 2]
```diff
@@ -672,3 +795,5 @@ After being created by an Indexer a healthy allocation goes through four states.
 - **Finalized** - Once an allocation has been closed there is a dispute period after which the allocation is considered **finalized** and it's query fee rebates are available to be claimed (claim()). The Indexer agent monitors the network to detect **finalized** allocations and claims them if they are above a configurable (and optional) threshold, **—-allocation-claim-threshold**.
 
 - **Claimed** - The final state of an allocation; it has run its course as an active allocation, all eligible rewards have been distributed and its query fee rebates have been claimed.
+
+Indexers are recommended to utilize offchain syncing functionality to sync subgraph deployments to chainhead before creating the allocation on-chain. This feature is especially useful for subgraphs that may take longer than 28 epochs to sync or have some chances of failing undeterministically.
```
[the remark this answers is not in the document]
Comment (hopeyen) [reply pages/en/network/indexing.mdx:799]: Oh SORRY, this review got a way from me somehow - will leave the updates for another PR. thank you and sorry again!

File: pages/en/network/indexing.mdx [hunk 3]
```diff
@@ -373,7 +371,7 @@ To successfully participate in the network requires almost constant monitoring a
 
 - **Indexer service** - The only component that needs to be exposed externally, the service passes on subgraph queries to the graph node, manages state channels for query payments, shares important decision making information to clients like the gateways.
 
-- **Indexer CLI** - The command line interface for managing the Indexer agent. It allows Indexers to manage cost models and indexing rules.
+- **Indexer CLI** - The command line interface for managing the Indexer agent. It allows Indexers to manage cost models, manual allocations, actions queue, and indexing rules.
```
[the remark this answers is not in the document]
Comment (hopeyen) [reply pages/en/network/indexing.mdx:374]: Making a note here that manual allocations is available after v0.19.3 and actions queue is v0.20.0

**Pull request merged**